pylint/checkers/utils.py

```diff
@@ -1476,6 +1480,43 @@ def is_registered_in_singledispatch_function(node: nodes.FunctionDef) -> bool:
     return False
 
 
+def find_infered_fn_from_register(node: nodes.NodeNG) -> nodes.FunctionDef | None:
+    # func.register are function calls or register attributes
+    # when the function is annotated with types
```

Comment:
Docstring instead of comment?

Reply:
There's another similar function that we decided to not yet modify to simplify the review of this pull request, I'll be creating a follow-up pull request that will combine some common functionality between the function that finds `singledispatch` and `singledispatchmethod`. That function also has a similar comment for function calls.

Should I add a proper docstring as part of the followup PR and I'll combine common functionality between both functions?

Reply:
That sounds good to me 👌